File: CONEXIONES/JAVA/MONGOconex.java
import com.mongodb.MongoClient;
import com.mongodb.MongoClientURI;

public class MONGOconex {
    public static void main(String[] args) {
        MongoClientURI uri = new MongoClientURI("mongodb://localhost:27017");
        MongoClient mongoClient = new MongoClient(uri);

        System.out.println("Conexión exitosa");

        mongoClient.close(); 
}

File: CONEXIONES/JAVASCRIPT/MONGOconex.js
const { MongoClient } = require('mongodb');

const uri = "mongodb://localhost:27017";
const client = new MongoClient(uri);

async function run() {
    try {
        await client.connect();
        console.log("Conexión exitosa");
    } catch (err) {
        console.error(err);
    } finally {
        await client.close();
    }
}

run().catch(console.dir);